composer.json

```
@@ -11,6 +11,8 @@
     ],
     "require": {
         "php": "^7.3",
+        "ext-json": "*",
+        "composer-plugin-api": "^1.1 || ^2.0",
```

Change to require Composer 2.0, goes together with change to the API usage:

```suggestion
        "composer-plugin-api": "^2.0",
```